query.go

```diff
@@ -37,7 +40,7 @@ func (fb *Firebase) EndAt(value string) *Firebase {
 func (fb *Firebase) OrderBy(value string) *Firebase {
 	c := fb.copy()
 	if value != "" {
-		c.params.Set(orderByParam, value)
+		c.params.Set(orderByParam, fmt.Sprintf("%q", value))
```

Comment:
I have one last concern about this. I agree that we should be automatically quoting the values being passed, there could be projects out there that are already using this and quoting the values.

It might be good to do something like:

```go
func (fb *Firebase) OrderBy(value string) *Firebase {
	c := fb.copy()
	if value != "" {
		value = fmt.Sprintf("%q", strings.Trim(value, `"`))
		c.params.Set(orderByParam, value)
	} else {
		c.params.Del(orderByParam)
	}
	return c
}
```

That way we can quote it only if someone doesn't pass a quoted string in.